node/src/chain_spec.rs

```diff
@@ -1,11 +1,22 @@
-use node_template_runtime::TuxedoGenesisConfig;
+use hex_literal::hex;
+use node_template_runtime::{
+    kitties::{KittyData, Parent},
+    money::Coin,
+    OuterConstraintChecker, OuterConstraintCheckerInherentHooks, OuterVerifier, WASM_BINARY,
+};
 use sc_service::ChainType;
+use tuxedo_core::{
+    genesis::TuxedoGenesisConfig,
+    inherents::InherentInternal,
+    verifier::{SigCheck, ThresholdMultiSignature, UpForGrabs},
+};
 
 // The URL for the telemetry server.
 // const STAGING_TELEMETRY_URL: &str = "wss://telemetry.polkadot.io/submit/";
 
 /// Specialized `ChainSpec`. This is a specialization of the general Substrate ChainSpec type.
-pub type ChainSpec = sc_service::GenericChainSpec<TuxedoGenesisConfig>;
+pub type ChainSpec =
+    sc_service::GenericChainSpec<TuxedoGenesisConfig<OuterVerifier, OuterConstraintChecker>>;
```

Comment:
Consider making a public type alias `type GenesisConfig = TuxedoGenesisConfig<OuterV, OuterC>;` in the runtime. We do this also for Transaction, Block, and I think other Tuxedo core types.

Basically the idea is that the Runtime provides the fully concrete types.